src/main/webapp/app/admin/logfile/logfile.component.html

```diff
@@ -0,0 +1,13 @@
+<div>
+    <h2>View logs</h2>
+
+    <jhi-route-selector></jhi-route-selector>
+
+    <div class="to-bottom" *ngIf="logtxt" (click)="scrollToBottom()">&#8675;Bottom</div>
```

Comment:
why dont you use some nice icon from font awesome here? same for to-top as well `arrow-circle-up` and `arrow-circle-down` would be good choice